app/models/message.rb

```diff
@@ -94,4 +95,26 @@ def send_telegram_message
       self.blocked = true
     end
   end
+
+  def send_facebook_message
+    return unless recipient&.facebook_id
+    return unless Rails.configuration.facebook_page_id
+
+    begin
+      Facebook::Messenger::Bot.deliver(
+        {
+          recipient: {
+            id: recipient.facebook_id
+          },
+          message: {
+            text: text
+          },
+          messaging_type: Facebook::Messenger::Bot::MessagingType::UPDATE
+        },
+        page_id: Rails.configuration.facebook_page_id
+      )
+    rescue Facebook::Messenger::Bot::SendError
+      # ignore
```

Comment:
This should better be handled in a application job which is OK to fail. Right now, we do everything in a single request - if one message cannot be send because the user or facebook has blocked us, then all subsequent actions wouldn't be executed either. Spawning a job for each message would make this error handling obsolete.